geolocator_android/android/src/main/java/com/baseflow/geolocator/MethodCallHandlerImpl.java

```diff
@@ -158,7 +158,7 @@ private void getLocationAccuracy(MethodChannel.Result result, Context context) {
             context,
             (ErrorCodes errorCode) ->
                 result.error(errorCode.toString(), errorCode.toDescription(), null));
-    if (result != null) {
+    if (status != null) {
```

Comment:
The check should be on both parameters as `result` can also be `null`:

```suggestion
    if (status != null && result != null) {
```

Reply:
Yea I had that at first, but this is not being checked anywhere else either.
If `result` can be null, then this will error everywhere in the file (see line above).

And also I don't think it can be null because of line 56:
```java
public void onMethodCall(@NonNull MethodCall call, @NonNull MethodChannel.Result result) {
```